.travis.yml

```diff
@@ -4,7 +4,12 @@ addons:
     apt:
         packages:
             - libsuitesparse-dev
+env:
+  - PRE_INSTALL='cython scipy'
+  - PRE_INSTALL=scipy
 install:
-    - pip install cython scipy
+    - pip install 'setuptools>=18'
+    - pip install $PRE_INSTALL
+    - pip list
```

Comment:
I merged the two lines 
```
pip install 'setuptools>=18'
pip install $PRE_INSTALL
```
and removed
```
pip list
```

But I have to use env, otherwise I will not have the tests once with `cython` installed and once without.